js/client-mainmenu.js

```diff
@@ -153,7 +153,7 @@
 			var $lastMessage = $chat.children().last();
 			var textContent = $lastMessage.html().indexOf('<span class="spoiler">') >= 0 ? '(spoiler)' : $lastMessage.children().last().text();
 			if (textContent && app.curSideRoom && app.curSideRoom.addPM && Tools.prefs('inchatpm')) {
```

Comment:
Hm, do we even need `textContent` and `$lastMessage` at all, if we do it this way?